### PR TITLE
Drop redundant Tanzu from TanzuClusterBootstrapTemplate and TanzuClus…

### LINKED ISSUE #1637: TAG review updates for resources
* Drop `Tanzu` from names TanzuClusterBootstrapTemplate and TanzuClusterBootstrap
* Need short names for KappControllerConfig, AntreaConfig, CalicoConfig

### PR DESCRIPTION
**Pull request by vijaykatam**
…terBootstrap

* The group name already has tanzu making the prefix redundant
* Add short names for KappControllerConfig, AntreaConfig and CalicoConfig

Fixes #1637

Signed-off-by: Vijay Katam <vkatam@vmware.com>

### Which issue(s) this PR fixes

Fixes #1637

### Describe testing done for PR

Envtests

### Release note

```release-note
package-based-lcm: Drop redundant Tanzu from TanzuClusterBootstrapTemplate and TanzuClusterBootstrap
```

### PR Checklist

- [x] Squash the commits into one or a small number of logical commits
      
- [x] Use good commit [messages](https://github.com/vmware-tanzu/tanzu-framework/blob/main/CONTRIBUTING.md)
- [x] Ensure PR contains terms all contributors can understand and links all contributors can access

### CODE REVIEW

File: addons/controllers/clusterbootstrap_handlers.go [hunk 1]
```diff
@@ -46,18 +46,18 @@ func (r *TanzuClusterBootstrapReconciler) TKRToClusters(o client.Object) []ctrl.
 	return util.ClustersToRequests(clusters, log)
 }
 
-func (r *TanzuClusterBootstrapReconciler) TanzuClusterBootstrapToClusters(o client.Object) []ctrl.Request {
-	bootstrap, ok := o.(*runtanzuv1alpha3.TanzuClusterBootstrap)
+func (r *ClusterBootstrapReconciler) ClusterBootstrapToClusters(o client.Object) []ctrl.Request {
+	bootstrap, ok := o.(*runtanzuv1alpha3.ClusterBootstrap)
 	if !ok {
 		r.Log.Error(errors.New("invalid type"),
 			"Expected to receive TanzuClusterBootstrap resource",
```
Comment (shivaani0505) [inline addons/controllers/clusterbootstrap_handlers.go:53]: TanzuClusterBootstrap should be changed here

File: addons/controllers/clusterbootstrap_controller.go [hunk 2]
```diff
@@ -149,13 +149,13 @@ func (r *TanzuClusterBootstrapReconciler) Reconcile(ctx context.Context, req ctr
 }
 
 // reconcileNormal reconciles the TanzuBootstrapCluster object
```
Comment (blc1996) [inline addons/controllers/clusterbootstrap_controller.go:151]: TanzuBootstrapCluster in comment

File: addons/controllers/clusterbootstrap_controller_test.go [hunk 3]
```diff
@@ -55,22 +55,22 @@ var _ = Describe("TanzuClusterBootstrap Reconciler", func() {
 
 		BeforeEach(func() {
 			clusterName = "test-cluster-tcbt"
-			clusterResourceFilePath = "testdata/test-cluster-tanzu-cluster-bootstrap.yaml"
+			clusterResourceFilePath = "testdata/test-cluster-cluster-bootstrap.yaml"
 		})
 
 		It("Should create clone TanzuClusterBootstrapTemplate and its related objects for the cluster and create package secret for foobar.example.com.1.17.2", func() {
```
Comment (blc1996) [inline addons/controllers/clusterbootstrap_controller_test.go:61]: TanzuClusterBootstrapTemplate in comment

File: addons/controllers/clusterbootstrap_controller_test.go [hunk 4]
```diff
@@ -96,9 +96,9 @@ var _ = Describe("TanzuClusterBootstrap Reconciler", func() {
 
 			// Verify providerRef exists and also the cloned provider object with ownerReferences to cluster and  TanzuClusterBootstrap
```
[the remark this answers is not in the document]
Comment (blc1996) [reply addons/controllers/clusterbootstrap_controller_test.go:97]: +1

File: addons/controllers/suite_test.go [hunk 5]
```diff
@@ -205,11 +205,11 @@ var _ = BeforeSuite(func(done Done) {
 		Scheme: mgr.GetScheme(),
 	}).SetupWithManager(ctx, mgr, controller.Options{MaxConcurrentReconciles: 1})).To(Succeed())
 
-	tanzuBootstrapReconciler := NewtanzuClusterBootstrapReconciler(mgr.GetClient(),
+	bootstrapReconciler := NewClusterBootstrapReconciler(mgr.GetClient(),
 		ctrl.Log.WithName("controllers").WithName("TanzuClusterBootstrap"),
```
Comment (blc1996) [inline addons/controllers/suite_test.go:209]: .WithName("TanzuClusterBootstrap"),

File: addons/controllers/clusterbootstrap_controller_test.go (hunk 3 above)
Comment (shivaani0505) [inline addons/controllers/clusterbootstrap_controller_test.go:61]: Change TanzuClusterBootstrapTemplate => ClusterBootstrapTemplate

File: addons/controllers/suite_test.go (hunk 5 above)
Comment (shivaani0505) [inline addons/controllers/suite_test.go:209]: Change in WithName("TanzuClusterBootstrap")

File: addons/controllers/clusterbootstrap_controller_test.go [hunk 6]
```diff
@@ -55,22 +55,22 @@ var _ = Describe("TanzuClusterBootstrap Reconciler", func() {
 
 		BeforeEach(func() {
 			clusterName = "test-cluster-tcbt"
-			clusterResourceFilePath = "testdata/test-cluster-tanzu-cluster-bootstrap.yaml"
+			clusterResourceFilePath = "testdata/test-cluster-cluster-bootstrap.yaml"
```
Comment (blc1996) [inline addons/controllers/clusterbootstrap_controller_test.go:58]: Nit: The name `test-cluster-cluster-bootstrap.yaml` looks a bit weird.

Comment (shivaani0505) [reply addons/controllers/clusterbootstrap_controller_test.go:58]: +1

**Review by shivaani0505 — APPROVED** [review]
LGTM

**Review by danniel1205 — APPROVED** [review]
LGTM

**Pull request merged**